sphinx/tutorial/rose/furthertopics/rose-bunch.rst

```diff
@@ -173,11 +175,11 @@ Naming Invocations
 
 While the different invocations of the command have their own output directed
 to indexed files, it can sometimes be difficult to quickly identify which file
-to look in for output. To aid this, ``rose_bunch`` supports naming command
-instances via the ``[bunch]names=`` option.
+to look in for output. To aid this, :rose:app:`rose_bunch` supports naming
+command instances via the :rose:conf:`rose_bunch[bunch]names=`` option.
```

Comment:
There's an extra backtick on the end of this `:ref:`, now appearing in the docs text.

Reply:
Yep - A second backtick at the end of a reference `ref` or domain `rose:` entry is always erroneous!